src/layer/vector/Polyline.js

```diff
@@ -165,11 +171,17 @@ export var Polyline = Path.extend({
 	// the polyline in case of a multi-polyline, but can be overridden by passing
 	// a specific ring as a LatLng array (that you can earlier access with [`getLatLngs`](#polyline-getlatlngs)).
 	addLatLng: function (latlng, latlngs) {
+		var oldLatLngs = this._convertLatLngs(this._latlngs);
```

Comment:
This is not good for performance, because it loops through all to copy it (and break reference). Is there a better solution?

Reply:
The purpose of `_convertLanlngs` is validating user-provided data.
We just do not need it here, I suppose.

Reply:
I used it to break the reference. If I use `this._latlngs` for `oldLatLngs` the added LatLng is added too, because it has the same reference.

Reply:
> I used it to break the reference.

Make separate light-weight function, based on `L.extend`.